react/src/lib/components/DeckGLMap/layers/terrain/map3DLayer.ts

```diff
@@ -285,12 +296,6 @@ export interface Map3DLayerProps<D> extends ExtendedLayerProps<D> {
     // Url to png image for map properties. (ex, poro or perm values as a texture)
     propertyTexture: string;
 
-    // Rotates map counterclockwise in degrees around 'rotPoint' specified below.
-    rotDeg: number;
```

Comment:
This prop and possible `rotPoint` is used in some contexts, so we should keep it but deprecate it, similarly the bounds prop.